tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "outDir": "./lib/",
```

Comment:
This file should keep `lib` as output directory. It is only used for developing the framework (every package of Foal is built into a `lib/` directory). The `tsconfig` file generated by the CLI for a new project is `packages/cli/src/generate/specs/app/tsconfig.json` (already updated in this PR).